src/test/java/org/fundacionjala/coding/kevincristhian/CesarCryptoTest.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.kevincristhian;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * test.
+ */
+public class CesarCryptoTest {
+
+    private static final String NEGATIVE_THREE = "-3";
+    private static final String NEGATIVE_FIVE = "-5";
+    private static final String THREE = "3";
+    private static final String FIFTY_TWO = "-52";
+    private static final String ZERO = "0";
+    private static final String FOX = "ZORRO";
+
+    /**
+     * decode encode cesar tests.
+     */
+    @Test
+    public void decode() {
+        CesarCrypto cesarCrypto = new CesarCrypto();
+        Assert.assertEquals("HOLA", cesarCrypto.decode("KROD", NEGATIVE_THREE));
+        Assert.assertEquals(FOX, cesarCrypto.decode("ETWWT", NEGATIVE_FIVE));
+        Assert.assertEquals("", cesarCrypto.decode("", THREE));
+        Assert.assertEquals("KROD", cesarCrypto.decode("HOLA", THREE));
+        Assert.assertEquals("HOL A", cesarCrypto.decode("HOL A", ZERO));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "HOL A" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)